cmd/transflect-operator/auxservers.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rs/zerolog/log"
+)
+
+type httpServer struct {
+	http.Server
+	name string
+}
+
+func newProbesServer(addr string) *httpServer {
+	ok := func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "OK")
+	}
+	mux := http.NewServeMux()
+	mux.HandleFunc("/_liveness", ok)
+	mux.HandleFunc("/_readiness", ok)
+	return &httpServer{
+		Server: http.Server{
+			Addr:    addr,
+			Handler: mux,
+		},
+		name: "probes",
+	}
+}
+
+func newMetricsServer(addr string) *httpServer {
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", promhttp.Handler())
+	return &httpServer{
+		Server: http.Server{
+			Addr:    addr,
+			Handler: mux,
+		},
+		name: "probes",
```

Comment:
Is this meant to have the same name as the other server, or should this be "metrics"?

Reply:
oups, sorry, thanks. done.